cypress/integration/Case07_Send_money.js

```diff
@@ -73,11 +15,13 @@ describe('Test case 7: Ability to send money', () => {
       LoginPage.mnemonicsInput.type(mnemonic)
       LoginPage.recoverWalletButton.click()
       LoginPage.yayButton.click()
-
-      HomePage.claimButton.click()
-
-      cy.request('POST', 'https://good-server.herokuapp.com/admin/queue', [ { password: 'password' }, { allow: 1 }, ])
-      // .then(response => { expect(response.body).to.have.property('ok', 0) })
+      HomePage.claimButton.click().invoke('text').then(text => {
+        cy.log(text)
+        if (text == 'Queue') {
+          const urlRequest = Cypress.env('postUrlRequest')
+          const bodyRequest = Cypress.env('postBodyRequest')
```

Comment:
add env var for the gundb password only, not for whole payload

call it using underscored capitals as other env vars (e.g. `CYPRESS_GUNDB_PASSWORD`)